src/dashboard/src/components/administration/views.py

```diff
@@ -223,6 +227,127 @@ def storage(request):
     system_directory_description = 'Available storage'
     return render(request, 'administration/locations.html', locals())
 
+def usage(request):
+    usage_dirs = _usage_dirs()
+
+    return render(request, 'administration/usage.html', locals())
+
+def _usage_dirs(calculate_usage = True):
+    # Get shared directory path
+    clientConfigFilePath = '/etc/archivematica/MCPClient/clientConfig.conf'
+    config = ConfigParser.SafeConfigParser()
+    config.read(clientConfigFilePath)
+
+    shared_dir = config.get('MCPClient', 'sharedDirectoryMounted')
+
+    # Get DIP uploads directory
+    dip_upload_dir = os.path.join(shared_dir, 'watchedDirectories/uploadDIP')
```

Comment:
This should be uploadedDIPs.  uploadDIP is the watched directory where we ask where the DIP should be uploaded to.  uploadedDIPs is where DIPs that have already been uploaded go to.  Also, splitting the path components given to os.path.join is preferred.